src/SaveObject.ts

```diff
@@ -364,7 +363,7 @@ function evaluateVersionCompatibility(ver: string | number): void {
   if (typeof ver !== "number") return;
   if (ver < 2) {
     AwardNFG(10);
-    initAugmentations();
+    Player.reapplyAllAugmentations();
```

Comment:
I'm 90% sure this was depending on initAugmentations() to set up factions. You should test with some of the old saves that were reported during the 2.3 transition to see if they will work.

Reply:
Wait, I had it turned around in my head. You're probably covered. Still, checking the old saves is a good idea.

Reply:
Yeah it looks like that's why it was being called there, not for the Player effect. That version probably had a lot of changes to faction augs (probably added a bunch of factions), and running that was the only way to get those augments to show up on the loaded factions without doing a soft reset first.